x-pack/plugin/esql/qa/testFixtures/src/main/resources/stats.csv-spec

```diff
@@ -2681,7 +2681,7 @@ c2:l |c2_f:l |m2:i |m2_f:i |c:l
 simpleCountOnFieldWithFilteringAndNoGrouping
 required_capability: per_agg_filtering
 from employees
-| stats c1 = count(emp_no) where emp_no < 10042
+| stats c1 = count(hire_date) where emp_no < 10042
```

Comment:
Hm, maybe it'd be better to add a new test and leave this alone - just so we have more edge case coverage?

Moreover, I think it'd be great to have a test where the field that's counted has missing/null and multi values - because otherwise, the query might as well use `count(*)` and disregard the field name.